src/components/admin/PortalAddEditForm.tsx

```diff
@@ -254,11 +251,12 @@ export const PortalAddEditForm: React.FC<PortalAddEditFormProps> = ({
         subtitle="(overrides general)"
       >
         <Checkbox
-          toggler
+          // toggler
```

Comment:
If this is needed for the future, it can be added back, and a well placed `@debt` or `@todo` comment would do, otherwise
```suggestion
```